### ISSUE TITLE
geolib breaks TypeScript projects with `noImplicitAny` compilation flag.

### ISSUE THREAD

**Issue by bkon** [manuelbieh/Geolib#144]
If project dependent on `geolib` has `noImplicitAny` TS compilation flag enabled, including geolib results in the following error:
 ```
ERROR in [at-loader] ./node_modules/geolib/dist/geolib.d.ts:195:14
    TS7010: 'convertUnit', which lacks return-type annotation, implicitly has an 'any' return type.

ERROR in [at-loader] ./node_modules/geolib/dist/geolib.d.ts:210:14
    TS7010: 'convertUnit', which lacks return-type annotation, implicitly has an 'any' return type
```

Geolib 2.0.23

**Comment by CodeAnimal** [comment]
I have a PR #146 that should fix this.

**Comment by bkon** [comment]
Solved with #146
